packages/@okta/vuepress-site/code/javascript/okta_sign-in_widget/index.md

```diff
@@ -139,6 +139,7 @@ In this case, you would like to use the Widget to sign in to a simple web page a
     <script type="text/javascript">
       const oktaSignIn = new OktaSignIn({
         baseUrl: "https://${yourOktaDomain}",
+        redirectUri: '{{redirectUri configured in OIDC app}}',
```

Comment:
Can we change this to `redirectUri configured in **your** OIDC app` to be more specific?